packages/pwa-kit-dev/bin/pwa-kit-dev.js

```diff
@@ -249,7 +249,7 @@ const main = async () => {
         .addOption(
             new program.Option(
                 '-b, --buildDirectory <buildDirectory>',
-                'a custom project directory where your build is located'
+                'a custom project build directory that you want to push'
```

Comment:
I feel like this description was a little misleading, and clarified that it's the build folder that we want.